### PR TITLE
Fix 54 update ci pipeline to use GitHub actions

### PR DESCRIPTION
**Pull request by oliverhamburger**

### CODE REVIEW

File: examples/sgp4Prop/unit_test/op_testrunner.c [hunk 1]
```diff
@@ -35,8 +35,11 @@ int main(void)
 {
     OP_App_Test_AddTestCases();
     return UtTest_Run();
-} /* End of main() */
+} 
+
+/* End of main() */
```
Comment (SpaceSteve121) [inline examples/sgp4Prop/unit_test/op_testrunner.c]: Can you revert these changes to these unrelated files? I'm seeing two files.

**Comment by SpaceSteve121** [comment]
Otherwise, looks good. 

Should we have seen the pipeline run for this branch? I'm not seeing anything in the UI that indicates it did or that it passed...

File: examples/sgp4Prop/unit_test/op_testrunner.c (hunk 1 above)
Comment (oliverhamburger) [reply examples/sgp4Prop/unit_test/op_testrunner.c]: Reverted one file that I added a couple extra blank lines, the other file I re-commented a main function that was supposed to be commented out, may have been left out from past pull request.

**Comment by oliverhamburger** [comment]
You can find the pipeline runs [here](https://github.com/oliverhamburger/ECI/actions)

File: examples/sgp4Prop/fsw/src/TestSGP4.c [hunk 2]
```diff
@@ -138,7 +138,7 @@ void runVER(VERIN *verins, int cnt)
         fclose(in_file);
     }
 }
-
+/*
 int main(void)
```
Comment (oliverhamburger) [inline examples/sgp4Prop/fsw/src/TestSGP4.c:142]: This change is reverting back to the init commit, looks like I made some small changes in the last pull request but now its back to where it was.
